lib/gui/pages/main/templates/main.tpl.html

```diff
@@ -89,7 +89,7 @@
           percentage="main.state.getFlashState().percentage"
           striped="{{ main.state.getFlashState().type == 'check' }}"
           ng-attr-active="{{ main.state.isFlashing() }}"
-          ng-click="flash.flashImageToDrive(main.selection.getImagePath(), main.selection.getDrive())"
+          ng-click="flash.flashImageToDrive(main.selection.getImage(), main.selection.getDrive())"
```

Comment:
I wonder if the specific changes related to this change should be refactored into their own mini-PR, and then rebase this PR once that mini-PR has been merged?
(Or perhaps I'm just being overly pedantic :-/  )